nixos/modules/services/backup/restic-rest-server.nix

```diff
@@ -84,6 +90,7 @@ in
         ExecStart = ''
           ${cfg.package}/bin/rest-server \
           --path ${cfg.dataDir} \
+          --htpasswd-file ${cfg.htpasswd-file} \
```

Comment:
@gabevenberg I think this fails if htpasswd-file is null because it will try to convert it to a string.

Reply:
Already fixed on master.